recipes/climlab/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "climlab" %}
+{% set version = "0.5.3" %}
+{% set sha256 = "ecd81c84af291e83cebfacd096cb13e7c6d1fa45401e969900f5934c2dd2fb9f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - export LDFLAGS="-undefined dynamic_lookup -bundle"  # [osx]
+    - python setup.py install
+
+requirements:
+  build:
+    - python 2.7*
+    - numpy x.x
+    - toolchain
+    - gcc
+    - libgfortran
```

Comment:
`gcc` brings `libgfortran` so there is no need to specify it here. (It is a run time dependency only.)